wemake_python_styleguide/violations/consistency.py

```diff
@@ -438,17 +438,18 @@ class MultipleInComparisonViolation(ASTViolation):
         Bring a consistency to the comparison!
 
     Solution:
-        Refactor your comparison expression.
-        Use different Comparisons.
+        Refactor your comparison expression to use several `and` conditions
```

Comment:
We use double `` signs to be rendered correctly by the `rst`.

Reply:
Ok